packages/grafana-ui/src/components/ConfirmButton/ConfirmButton.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { ConfirmButton } from './ConfirmButton';
+import { shallow } from 'enzyme';
+
+describe('ConfirmButton', () => {
+  let wrapper: any;
+  let deleted: any;
+
+  beforeAll(() => {
+    deleted = false;
```

Comment:
Instead of introducing this flag you can use Jest's function mock anf assert the function was actually called rather than asserting this value was updated

Reply:
It's just a copy of tests from DeleteButton component, so it needs to be refactored.